recipes/boost/bld.bat

```diff
@@ -0,0 +1,48 @@
+:: Set the right msvc version according to Python version
+REM write a temporary batch file to map cl.exe version to visual studio version
+echo @echo 15=9 > msvc_versions.bat
+echo @echo 16=10 >> msvc_versions.bat
+echo @echo 17=11 >> msvc_versions.bat
+echo @echo 18=12 >> msvc_versions.bat
+echo @echo 19=14 >> msvc_versions.bat
+
+REM Run cl.exe to find which version our compiler is
+for /f "delims=" %%A in ('cl /? 2^>^&1 ^| findstr /C:"Version"') do set "CL_TEXT=%%A"
+FOR /F "tokens=1,2 delims==" %%i IN ('msvc_versions.bat') DO echo %CL_TEXT% | findstr /C:"Version %%i" > nul && set VSTRING=%%j && goto FOUND
+EXIT 1
+:FOUND
+
+call :TRIM VSTRING %VSTRING%
+
+:: Start with bootstrap
+call bootstrap.bat
+if errorlevel 1 exit 1
+
+:: Build step
+.\b2 install ^
```

Comment:
You may be right. Let me work on the test part. I'll make sure this does a comprehensive test on the libs for all platforms. That way, it will be clear what (if any) other flags need to be added here.